### PR TITLE
Add details for customs import

### PR DESCRIPTION
**Pull request by mkushe**
Add details for VOEC (VAT on Ecommerce) field to be added in booking request for customs import to Norway.

### CODE REVIEW

**Comment by bringdev** [comment]
[trello.com/c/9KZZ7HR1](https://trello.com/c/9KZZ7HR1)

File: api/booking/api.raml [hunk 1]
```diff
@@ -166,6 +166,44 @@ documentation:
...
+
+    ### Example
+
+    In this example, you are sending a parcel/shipment with 2 shampoos, 5 cotton t-shirts and 1 wool jacket. The number of pieces is 8 (2 shampoos + 5 cotton
+    t-shirts + 1 wool jacket), and the number of articles is 3 (shampoo, cotton t-shirt, wool jacket). The whole shipment is referred to as an item.
+
+    | Quantity | Description of contents | Customs tariff code | Total content net weight (kg) | Total content value | Currency | Country of origin |
+    |:--------:|:-----------------------:|:-------------------:|:----------------------------:|:-------------------:|:--------:|:-----------------:|
+    | `2` | `shampoo` | `33.05.1000` | `3.0` | `200` | `NOK` | `CH`|
+    | `5` | `cotton t-shirts` | `61.09.1000` | `2.5` | `1000`| `NOK`| `PL` |
```
Comment (tomgag) [inline api/booking/api.raml:202]: Ah, the dreaded shampoo and t-shirts again, sigh...

File: api/booking/examples/customsImportDeclaration.json [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+              "quantity": 1,
+              "goodsDescription": "wool jacket",
+              "customsArticleNumber": "61.04.3100",
+              "itemNetWeightInKg": 2.0,
+              "tarriffLineAmount": 750,
+              "currency": "NOK",
+              "countryOfOrigin": "DE"
+            }
+          ],
+          "vatOnEcommerce": "1234567"
```
Comment (tomgag) [inline api/booking/examples/customsImportDeclaration.json:72]: should we add the nature of transaction or whatever it's called here, too?

File: api/booking/examples/customsImportDeclaration.xml [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+            <email>tore@mottakertest.no</email>
+            <phoneNumber>+4799999999</phoneNumber>
+          </contact>
+        </recipient>
+      </parties>
+      <product>
+        <id>HOME_DELIVERY_PARCEL</id>
+        <customerNumber>PARCELS_NORWAY_INTERNATIONAL-1045151</customerNumber>
+        <ediCustomsDeclarations>
+          <vatOnEcommerce>1234567</vatOnEcommerce>
```
Comment (tomgag) [inline api/booking/examples/customsImportDeclaration.xml:42]: same here

**Review by tomgag — APPROVED** [review]
LGTM 👍

File: api/booking/examples/customsImportDeclaration.json (hunk 2 above)
Comment (mkushe) [reply api/booking/examples/customsImportDeclaration.json:72]: its not required in case of import

File: api/booking/api.raml (hunk 1 above)
Comment (mkushe) [reply api/booking/api.raml:202]: whatever we Exported, importing that only :)

**Pull request merged**